website/pages/blog/aws-encryption-and-multi-account-access.md

```diff
@@ -0,0 +1,270 @@
+
+![AWS Managed key: aws/s3](/images/blog/aws-encryption-and-multi-account-access/aws-managed-key-s3.png)
+*AWS Managed key: aws/s3*
+
+We’ll modify the encryption setting for the S3 bucket in the example data AWS Account to use the aws/s3 AWS managed KMS key and then upload a new object which will use that aws/s3 encryption setting and encrypt the object with the aws/s3 AWS managed KMS key.
+
+![AWS Default Encryption Settings for S3 Bucket](/images/blog/aws-encryption-and-multi-account-access/s3-default-encryption.png)
+*AWS Default Encryption Settings for S3 Bucket*
+
+Now, after ensuring that we’ve properly configured the IAM policies attached to the IAM role in the example compute AWS account and the S3 bucket policy attached to the bucket in the data AWS account to permit for access for the IAM role to the S3 bucket and objects, we’ll try to retrieve this new object encrypted with the aws/s3 AWS managed key.  We get an AccessDenied error with the additional context that either the key doesn’t exist or we’re not allowed to access the key.
+
+![AccessDenied error for cross-account S3 GetObject](/images/blog/aws-encryption-and-multi-account-access/not-allowed-to-access-key.png)
+*AccessDenied error for cross-account S3 GetObject*
+
+In this case, due to the key being an AWS Managed KMS Key, we’re unable to access the KMS Key due to the following KMS Key Policy.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Id": "auto-s3-2",
+    "Statement": [
+        {
+            "Sid": "Allow access through S3 for all principals in the account that are authorized to use S3",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "*"
+            },
+            "Action": [
+                "kms:Encrypt",
+                "kms:Decrypt",
+                "kms:ReEncrypt*",
+                "kms:GenerateDataKey*",
+                "kms:DescribeKey"
+            ],
+            "Resource": "*",
+            "Condition": {
+                "StringEquals": {
+                    "kms:CallerAccount": "123412341234",
+                    "kms:ViaService": "s3.us-east-1.amazonaws.com"
+                }
+            }
+        },
+        {
+            "Sid": "Allow direct access to key metadata to the account",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::123412341234:root"
+            },
+            "Action": [
+                "kms:Describe*",
+                "kms:Get*",
+                "kms:List*"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+Let’s take a deeper look at this key policy and what this statement does.  Since they’re both Allow statements, these 2 statements function as a logical or statement for access.  This Key Policy has 2 main statements:
```

Comment:
```suggestion
Let’s take a deeper look at this key policy and what this statement does.  Since they’re both Allow statements, these 2 statements function as a logical **or** statement for access.  This Key Policy has 2 main statements:
```